app/web/src/newhotness/ApprovalFlow.vue

```diff
@@ -235,9 +236,20 @@ const props = defineProps<{
   user: User;
 }>();
 
-const status = computed(
+// FIXME(nick): remove all pre-ReBAC stuff from this component and only use "satisfied" below to
+// determine if something has been approved. This component should only be used if the change set
+// is in "NeedsApproval" state, but that will require a small refactor.
```

Comment:
Originally in #6932, refactoring this stuff out helped uncover how these components truly worked. Distractions removed, etc. However, the end fixes ended up not being related, so they were not included in this PR.